share/zfsnap/core.sh

```diff
@@ -241,6 +269,57 @@ IsSnapshot() {
     esac
 }
 
+#Find most recent zfsnap snapshot in list of snapshots
+LatestSnap(){
+	local snaps=("$@")
+	
+	local latest_snap=$(echo "${snaps[*]}" | tr " " "\n" | grep $(echo ${snaps[*]} | tr " " "\n" | /bin/cut -d "-" -f4-6 | /bin/sort | /usr/bin/tail -n 1) | /usr/bin/tail -n 1 )
```

Comment:
This is a giga-on liner. How about `zfs list -d 1 -t snapshot -H -o name | tail -1`.

Also, as an aside, please don't hard code paths as many distros/OSs have... shall we say... different ideas about where stuff belongs. ;-)